inscriptions/ISic003020.xml

```diff
@@ -196,37 +211,34 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI332330">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                     ἐπὶ<lb n="2"/> ἀμφιπό-<lb n="3"/>λου ἐν Συ-
-                    <lb n="4"/>
-                    <supplied reason="lost">ρακο</supplied>ύσαις
-                    <supplied reason="lost">— —</supplied>
-                    <lb n="5"/>
-                    <supplied reason="lost">— — —</supplied>ΠΟ
-                    <supplied reason="lost">— — —</supplied>
-                    <lb n="6"/>
-                    <surplus>²versus 6-11 non lecti sunt</surplus>²<lb n="7"/>ΠΟ
-                    <supplied reason="lost">— — — — —</supplied>
-                    <lb n="8"/>ΓΕΝ
-                    <supplied reason="lost">— — — — —</supplied>
-                    <lb n="9"/>
-                    <supplied reason="lost">Ῥ</supplied>ουφίλ
-                    <supplied reason="lost">λα(?)</supplied>
-                    <lb n="10"/>Φαυστια
-                    <supplied reason="lost">νός(?)</supplied>
-                    <lb n="11"/>v Ἀρτέ
-                    <supplied reason="lost">μων(?)</supplied>
-                    <lb n="12"/> ΙΙΧΡΜΝ
-                    <supplied reason="lost">— — —</supplied>
-                    <lb n="13"/>
+                    <lb n="1"/><handShift new="#h1"/><supplied reason="lost" evidence="previouseditor">ἐπὶ</supplied>
```

Comment:
for previous editor, I think it should be supplied reason="undefined" evidence="previouseditor"